src/umleditor/mvc_view/gui_view/view_GUI.py

```diff
@@ -71,6 +77,9 @@ def confirm_class_clicked(self):
         """
         On Confirm emits signal to process task
         """
+        if self._size >= self._max_size:
+            self.invalid_input_message("No more than " + str(self._max_size) + " Class Cards in a single diagram!")
```

Comment:
for now this is okay but I think we should end up hoping to be able to support infinite with the knowledge that some will overlap. Adding a "hide detail" option would also help - condensing the three white blocks would allow users to fit many more.

Reply:
infinite meaning the only bound is memory size

Reply:
unless we want to do dynamic culling, kekW

Reply:
Tim notes that a scrollbar could also help